question-service/model/question-model.js

```diff
@@ -15,10 +15,6 @@ let QuestionModelSchema = new Schema({
     type: String,
     required: true,
   },
-  example: {
-    type: String,
-    required: true,
-  },
 });
 
 export default mongoose.model("QuestionModel", QuestionModelSchema);
```

Comment:
To discuss with the team on standardization of defining models.

Personally I would create the file as `question.js`, define the schema as `questionSchema`, and this line would be
`export default mongoose.model("Question", questionSchema); `

Reason is because mongoDB looks at "Question", and creates a collection in small letters that is plural, which is "questions" in this case. Hope this helps!